frontend/src/api/k8s/inferenceServices.ts

```diff
@@ -127,18 +127,9 @@ export const assembleInferenceService = (
 
   const dashboardNamespace = data.dashboardNamespace ?? '';
   if (!isModelMesh && podSpecOptions && podSpecOptions.selectedHardwareProfile) {
-    const isLegacyHardwareProfile =
-      !!podSpecOptions.selectedAcceleratorProfile ||
```

Comment:
@gage does this code need to be put back; except with the line above being:
`const isAcceleratorProfile = !! podSpecOptions.selectedAcceleratorProfile`